connector/subgraph/src/aragon-hooks.ts

```diff
@@ -3,11 +3,12 @@ import { loadAppConfig } from './helpers'
 
 
 const APP_IDS: string[] = [
-  '0x0fe6b8bdb08ec31cf72c32af3c168ea1f09de36414edebdf6b3b1b7970093680',  // conviction-voting.aragonpm.eth
-  '0xbc5e8545c829b4a2dd66039e0824a32c19e8159e699402865a9e18746f99c390',  // conviction-voting.1hive.aragonpm.eth
-  '0x589851b3734f6578a92f33bfc26877a1166b95238be1f484deeaac6383d14c38',  // conviction-voting.open.aragonpm.eth 
-  '0xe4691f497f5e74daf61612cea2d5a540b095805872218eaa9108aa5fd76779a2'   // conviction-beta.open.aragonpm.eth
-]   
+  // '0x0fe6b8bdb08ec31cf72c32af3c168ea1f09de36414edebdf6b3b1b7970093680',  // conviction-voting.aragonpm.eth 
+  // '0xbc5e8545c829b4a2dd66039e0824a32c19e8159e699402865a9e18746f99c390',  // conviction-voting.1hive.aragonpm.eth TODO: Should be removed when new conviction voting installed on xdai
+  // '0x589851b3734f6578a92f33bfc26877a1166b95238be1f484deeaac6383d14c38',  // conviction-voting.open.aragonpm.eth 
+  // '0xe4691f497f5e74daf61612cea2d5a540b095805872218eaa9108aa5fd76779a2',  // conviction-beta.open.aragonpm.eth 
```

Comment:
Commented temporarily since this versions have the older functionality and would cause the subgraph to fail.